packages/swingset-liveslots/tools/fakeVirtualSupport.js

```diff
@@ -167,6 +167,12 @@ export function makeFakeLiveSlotsStuff(options = {}) {
     return d && (weak ? d.deref() : d);
   }
 
+  function requiredValForSlot(slot) {
+    const val = getValForSlot(slot);
+    assert(val, `no value for ${slot}`);
```

Comment:
Ah, the perennial issue about modality of error messages.
```suggestion
    assert(val, `${slot} must have a value`);
```